### PR TITLE
Add changelog for 1.9.7 and 1.10.2

### PR DESCRIPTION
**Pull request by 5chdn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+## Parity [v1.10.2](https://github.com/paritytech/parity/releases/tag/v1.10.2) (2018-04-24)
+
+Parity 1.10.2 is a bug-fix release to improve performance and stability.
+
+The full list of included changes:
+
+- Update Parity beta to 1.10.2 + Backports ([#8455](https://github.com/paritytech/parity/pull/8455))
+  - Update Parity beta to 1.10.2
+  - Allow 32 bit pipelines to fail ([#8454](https://github.com/paritytech/parity/pull/8454))
+    - Disable 32bit tragets for gitlab
```
Comment (niklasad1) [inline CHANGELOG.md]: s/tragets/targets

Comment (niklasad1) [reply CHANGELOG.md]: s/32bit/32 bit

File: docs/CHANGELOG-1.9.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
+## Parity [v1.9.7](https://github.com/paritytech/parity/releases/tag/v1.9.7) (2018-04-23)
+
+Parity 1.9.7 is a bug-fix release to improve performance and stability.
+
+The full list of included changes:
+
+- Update Parity stable to 1.9.7 + Backports ([#8456](https://github.com/paritytech/parity/pull/8456))
+  - Update Parity stable to 1.9.7
+  - Allow 32 bit pipelines to fail ([#8454](https://github.com/paritytech/parity/pull/8454))
+    - Disable 32bit tragets for gitlab
```
Comment (niklasad1) [inline docs/CHANGELOG-1.9.md]: s/Disable 32bit tragets for gitlab/Disable 32 bit targets for gitlab

**Comment by 5chdn** [comment]
Thanks, you are totally right :)

**Pull request merged**